.github/workflows/CI.yaml

```diff
@@ -37,49 +37,40 @@ jobs:
     - name: Create Psi4 Environment
       uses: conda-incubator/setup-miniconda@v2
       with:
-        activate-environment: p4dev
+        activate-environment: p4env
         environment-file: devtools/conda-envs/psi.yaml
```

Comment:
do we even need this anymore? it is an empty env with `- nodefaults`. I feel like we could just create the env here and drop `psi.yaml` entirely

Reply:
I'm hesitant to remove it only because we may need to add options later as Psi4 evolves.